lib/round.rb

```diff
@@ -0,0 +1,69 @@
+class Round
+  attr_reader :deck, 
+              :turns, 
+              :current_turn, 
+              :number_correct, 
+              :correct_qs, 
+              :curr_card
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+    @current_turn = 0
+    @number_correct = 0
+    @correct_qs = []
+  end
+
+  def current_card
+    @curr_card = @deck.cards[@current_turn]
+  end
+
+  def take_turn(guess_string)
+   if @current_turn > @deck.count
+      return  "you have reached the limit of questions"
```

Comment:
I'm not sure I see where this ever gets used. The counter in your runner would prevent this from ever happening, I think. You can probably get rid of this `if` statement and just start this method with `@new_turn = Turn.new....`